### ISSUE TITLE
[ARM64] Trying to build ./b.sh --qt on Linux  

### ISSUE THREAD

**Issue by clort81** [hrydgard/ppsspp#14206]
On Linux (Devuan Ceres), aarch64, fresh recursive clone of git master from 2021-02-23:  
```
git clone --recurse-submodules https://github.com/hrydgard/ppsspp.git
```
b.sh gets through whole build (**awesome!**), but final link has one issue: 

```
$ ./b.sh --qt
[snip]
[100%] Linking CXX executable PPSSPPQt
/usr/bin/ld: ../ffmpeg/linux/aarch64/lib/libavformat.a(avienc.o): relocation R_AARCH64_ADR_PREL_PG_HI21 against symbol `__stack_chk_guard@@GLIBC_2.17' which may bind externally can not be used when making a shared object; recompile with -fPIC
```

Seems like rebuilding libavformat with -fPIC shouldn't be hard, or perhaps this is exposing some other problem that I'm not aware of, so I report this here.

**Comment by unknownbrackets** [comment]
Try editing `ffmpeg/linux_arm64.sh` and add:

```sh
--enable-pic \
```

Underneath:

```sh
--enable-zlib \
```

Then run:

```sh
pushd ffmpeg
./linux_arm64.sh
popd
```

Then try `b.sh` again.

-[Unknown]

**Comment by clort81** [comment]
This solved my build problem, thank you unknownbrackets!

Running ppsspp with OpenGL/Glew, SDL2/Xlib under Wayfire/Wayland.
Trails in the Sky is fullspeed, using only 22-29% cpu (!)

Device is Khadas Vim3pro (4x Cortex A73 @ 2.2ghz, mali G52 MP4 @ 800mhz, panfrost drivers)

It's beautiful to see.  Huge props to the developers!  So efficient! :)

**Comment by FridgeRacer** [comment]
Brackets' fix also works on Nintendo Switch, Switchroot Ubuntu 18.04 (bionic). The whole project builds and works as expected now. Thanks a bunch!

Also needed fix from #14545

Built with a recursive clone done at Jun 21 00:23 CEST
